client-sdk/go/modules/consensusaccounts/types.go

```diff
@@ -4,11 +4,13 @@ import "github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
 
 // Deposit are the arguments for consensus.Deposit method.
 type Deposit struct {
+	To     types.Address   `json:"to,omitempty"`
```

Comment:
This probably needs to be `*types.Address`.

Reply:
oh yeah probably